signinapp/model.py

```diff
@@ -164,13 +188,57 @@ def make(name: str, password: str, role: Role, preferred_name: str = None,
                     subteam=subteam,
                     approved=approved)
 
+    @staticmethod
+    def make_guardian(student: User, name: str, phone_number: str, email: str, preference: int = 1):
+        guardian_limted = Role.from_name("guardian_limited")
```

Comment:
`limted`?
You could also just call it `guardian_role`

Reply:
The idea is we would add another role for guardians that have actually registered, so this one is "limited". Could definitely use a better name though.

Reply:
I was talking about your variable name, not the role name